minecraft_servers/lendmark/manifest.json

```diff
@@ -1,15 +1,18 @@
 {
     "server_name": "lendmark",
-    "nice_name": "LendMark",
+    "nice_name": "LendMark Network",
     "direct_ip": "mc.lendmark.sk",
     "server_wildcards": [
-      "%.lendmark.sk",
-      "%.lend.games"
+      "%.lendmark.sk"
+    ],
+    "supported_languages": [
+        "sk",
+        "cz"
     ],
     "social": {
       "web": "https://lendmark.sk/",
       "web_shop": "https://store.lendmark.sk/",
-      "discord": "https://discord.com/invite/kaMfqnvaqC",
+      "discord": "https://discord.lendmark.sk",
```

Comment:
Please change this back to an official Discord invite link (referring to [docs/Manifest.md#discord-url](https://github.com/LabyMod/server-media/blob/master/docs/Manifest.md#discord-url)